.buildkite/pipeline.yml

```diff
@@ -62,23 +62,47 @@ steps:
           packages.baseDAO-ligo-meta.exes.baseDAO-ligo-meta
         -c bats ./scripts/test.bats
 
-  - label: test
+  - label: test-local-chain-007
+    env:
+      TASTY_NETTEST_NODE_ENDPOINT: "http://localhost:8735"
     if: *not_scheduled
-    commands:
+    commands: &test
     - nix-build ci.nix -A packages.baseDAO.tests.baseDAO-test
-    # Emulator + local chain for Delphi protocol
+    - export TASTY_NETTEST_DATA_DIR="$(mktemp -d --tmpdir="$PWD")"
     - nix run -f ci.nix tezos-client -c
       ./result/bin/baseDAO-test --nettest-run-network
-      -A localhost -P 8735
+
+  - label: test-local-chain-008
+    env:
+      TASTY_NETTEST_NODE_ENDPOINT: "http://localhost:8734"
+    if: *not_scheduled
+    commands: *test
 
   - label: ligo-test
+    env:
+      TASTY_NETTEST_NODE_ENDPOINT: "http://localhost:8735"
     if: *not_scheduled
-    commands:
+    commands: &ligo-test
     - nix-build ci.nix -A packages.baseDAO-ligo-meta.tests.baseDAO-test
     # Running only on emulator for now due to oversized operations
-    - nix run -f ci.nix tezos-client -c
-      ./result/bin/baseDAO-test --nettest-no-run-network
-      -A localhost -P 8735
+    - ./result/bin/baseDAO-test --nettest-no-run-network
+
+  # TODO [#124] Uncomment
+  # - label: ligo-test-local-chain-007
+  #   env:
+  #     TASTY_NETTEST_NODE_ENDPOINT: "http://localhost:8735"
+  #   if: *not_scheduled
+  #   commands: &ligo-test
+  #   - nix-build ci.nix -A packages.baseDAO-ligo-meta.tests.baseDAO-test
+  #   # Running only on emulator for now due to oversized operations
+  #   - nix run -f ci.nix tezos-client -c
+  #     ./result/bin/baseDAO-test --nettest-run-network
```

Comment:
Couldn't we have these un-commented but with the `--nettest-no-run-network` option instead?

Reply:
there is such a job above

Reply:
Wait, above there's the ones for the haskell version, aka `test-local-chain-007` and `test-local-chain-008` and one for the ligo version `ligo-test`.

My suggestion is to (aside from keeping the haskell version ones) to remove `ligo-test` and add `ligo-test-local-chain-007` and `ligo-test-local-chain-008` but with `--nettest-no-run-network` (and a `TODO [#124] change to nettest-run-network` just above that)

Reply:
Hmm, what is the reason to have two jobs with `--nettest-no-run-network`? They'll run the same set of tests twice :thinking:

Reply:
That's a fair point, I just want to make sure not to leave `ligo-test` behind.

How about replacing it with `ligo-test-local-chain-007` (`--nettest-no-run-network` with comment) and keep `ligo-test-local-chain-008` only fully commented out?

Reply:
Well, Ideally we should have 3 jobs:
1) _only_ emulator tests
2) _only_ 007 local-chain tests
3) _only_ 008 local-chain tests
This way it's currently done in `morley` repository

But I don't know how to do it in buildkite yet:(
See [this thread](https://forum.buildkite.community/t/escaped-shell-variable-in-pipeline-config/1451) on buildkite forum

Reply:
I agree, let's leave only `ligo-test-local-chain-007` for now with a comment that it doesn't actually run any network tests

Reply:
@pasqu4le, I've refactored test jobs once again to have the :proper: structure. Please, rereview once again:)